css-positioning/fixed.css

```diff
@@ -0,0 +1,11 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  left: 500px;
+  top: 100px;
```

Comment:
BIG movement here. Instead, you should set a `width: 100%` on the element so `text-align` can do its work.